ServerCore/Pages/Hints/Edit.cshtml

```diff
@@ -13,6 +13,10 @@
     <a asp-page="./Index">Cancel</a>
 </div>
 
+<div>
```

Comment:
Not sure why these hint things are part of this PR, but if you feel they're truly ready to go I won't quibble, in the name of throughput.

Reply:
There are no changes related to hints. Github confusion. I will fix.